e2e/e2e/packet.py

```diff
@@ -194,7 +194,7 @@ def query_unreceived_acks(
 def packet_send(c: Config, src: ChainId, dst: ChainId,
                 src_port: PortId, src_channel: ChannelId,
                 amount: int, height_offset: int, number_msgs: Optional[int] = None,
-                key: Optional[str] = None) -> Packet:
+                key: Optional[str] = 'user2') -> Packet:
```

Comment:
Is this right?

Reply:
not sure we want to commit this as it then depends on the `dev-env` to init gaiad or in general to have `user2`. The reason was to have the script using different key for `ft-transfer` than for `hermes start`. Otherwise we run into bad seq number.

Reply:
I think I see. This change was basically the reason we needed to adjust some of our CI scripts (https://github.com/informalsystems/ibc-rs/pull/1089/).

It makes sense to generate & maintain two sets of keys. Should we make this more explicit, however?
Changing the key names can go a long way in helping understand what's happening, e.g., `user` -> `relaying_user`, and `user2` -> `transfer_user`.

Reply:
I agree that we should change the key names to some more meaningful, but we don't have to it for this release as it's not a user-facing change.